### PR TITLE
Update message signing/verify with updated function from SDK

### LINKED ISSUE #5210: Update message signing/verify with updated function from SDK
### Description

As a result of an improvement to the signing function as described in this [LIP](https://github.com/LiskHQ/lips/pull/429). There is a need to update the sign and verify message functions been consumed from the SDK.

### Acceptance Criteria
- Update `cryptography.ed.verifyMessageWithPublicKey` with the improved verifyMessage function from SDK
- Update `cryptography.ed.signAndPrintMessage` with the improved sign message function from SDK
- Update unit tests

Please describe the conditions which must be met for this issue to close

### Additional Information
Related SDK  issue: https://github.com/LiskHQ/lisk-sdk/issues/8775#issue-1822439117

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #5210

### How was it solved?

Bump lisk-client

### How was it tested?

- The cryptography has breaking changes, hence you must remove all the accounts and add them back to test all scenarions related to encrypt and decrypt
- Send tokens to check for sanity

### CODE REVIEW

File: src/modules/message/store/action.js [hunk 1]
```diff
@@ -3,11 +3,19 @@ import { cryptography } from '@liskhq/lisk-client';
 import { signMessageUsingHW } from '@wallet/utils/signMessage';
 import { signMessageWithPrivateKey } from '../utils/signMessageWithPrivateKey';
 
+export const getUnsignedMessage = (message) =>
```
Comment (eniolam1000752) [inline src/modules/message/store/action.js]: I think we can go with 
```suggestion
export const getUnsignedNonProtocolMessage = (message) =>
```

File: src/modules/message/store/action.js [hunk 2]
```diff
@@ -3,11 +3,19 @@ import { cryptography } from '@liskhq/lisk-client';
...
+    Buffer.from(message, 'utf8'),
+  ]).toString('hex');
+
 export const signMessage =
   ({ nextStep, message, privateKey, currentAccount }) =>
   async () => {
     if (currentAccount?.hw) {
-      const [error, signature] = await to(signMessageUsingHW({ message, account: currentAccount }));
+      const [error, signature] = await to(
+        signMessageUsingHW({ account: currentAccount, message: getUnsignedMessage(message) })
```
Comment (eniolam1000752) [inline src/modules/message/store/action.js]:
```suggestion
        signMessageUsingHW({ account: currentAccount, message: getUnsignedNonProtocolMessage(message) })
```

File: src/modules/message/store/action.test.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { cryptography } from '@liskhq/lisk-client';
 import { mockHWAccounts } from '@hardwareWallet/__fixtures__';
 import * as signMessageUtil from '@wallet/utils/signMessage';
-import { signMessage } from './action';
+import { getUnsignedMessage, signMessage } from './action';
```
Comment (ManuGowda) [inline src/modules/message/store/action.test.js]:
```suggestion
import { getUnsignedNonProtocolMessage, signMessage } from './action';
```

File: src/modules/message/store/action.test.js [hunk 4]
```diff
@@ -47,7 +47,7 @@ describe('balanceReclaimed', () => {
     await signMessage({ nextStep, privateKey, message, currentAccount: mockCurrentAccount })();
 
     expect(signMessageUtil.signMessageUsingHW).toHaveBeenCalledWith({
-      message,
+      message: getUnsignedMessage(message),
```
Comment (ManuGowda) [inline src/modules/message/store/action.test.js]:
```suggestion
      message: getUnsignedNonProtocolMessage(message),
```

**Review by eniolam1000752 — APPROVED** [review]

**Pull request merged**